### PR TITLE
Enable bottom right corner drag resizer in VST2

### PR DESCRIPTION
**Pull request by mkruselj**

### CODE REVIEW

File: src/surge-xt/SurgeSynthEditor.cpp [hunk 1]
```diff
@@ -330,7 +331,7 @@ void SurgeSynthEditor::resized()
     if (zfn != 1.0 && rezoomGuard == 0)
     {
         auto br = BlockRezoom(this);
-        adapter->setZoomFactor(round(adapter->getZoomFactor() * zfn), false);
+        adapter->setZoomFactor(round(adapter->getZoomFactor() * zfn), true);
```
Comment (baconpaul) [inline src/surge-xt/SurgeSynthEditor.cpp:334]: Uhhh why this change?

Comment (mkruselj) [reply src/surge-xt/SurgeSynthEditor.cpp:334]: My mistake, I was trying stuff out and forgot to revert it. Apols!
